### ISSUE TITLE
Unable to obtain test files

### ISSUE THREAD

**Issue by dam-pav** [BotBlake/pyTAB#35]
### Checklist
- [x] Using the latest Version (branch `develop`) - old Versions aren't supported
I followed the steps specified at _https://github.com/BotBlake/jellybench_py?tab=readme-ov-file#installing-jellybench_py_
- [x] Running the Script on bare metal (no VM)
Running on Ubuntu Server 24.10
- [ ] Manually executing `python /jellybench_py/hwi.py` works
Running hwi.py (Hardware Wallet Interface?) returns this:
_Traceback (most recent call last):
  File "/home/damjan/jellybench_py/./jellybench_py/hwi.py", line 25, in <module>
    import cpuinfo
ModuleNotFoundError: No module named 'cpuinfo'_

What was it supposed to say? Actually, the running is done with python3 (instead of python), something like this  `python3 ./jellybench_py/hwi.py`, if that is all the same.

- [ ] The issue occurs AFTER the Benchmark has started
Of course.

### Description
This is as far as it gets:

```

Welcome to jellybench_py Cheeseburger Edition 🍔

Disclaimer
| Please close all background programs and plug the device into a power source if it is running on battery power before starting the
| benchmark.
Confirm [y/N]: y

System Initialization
| Fetch Supported Platforms... success!
| Obtaining System Information... success!
| Detected System Config:
|   OS: Ubuntu 24.10
|   CPU: Intel(R) N100
|     Threads: 4
|     Arch: x86_64
|   RAM:
|     - Generic 17045 mb 0
|   GPU(s):
|     1: Alder Lake-N [UHD Graphics]
| Loading tests...  success!
Done

Loading ffmpeg
Note: Note: No compatible hashing method found.
INFO: Replacing existing files with validated ones.
Unpacking Archive... success!
Done

Obtaining Test-Files:
| "jellyfish-40-mbps-hd-h264" - Error

The following Error occured: 404
Press any key to exit

```
### Steps to Reproduce
1.  Follow jellybench installation steps.
2.  Run poetry
3.  start jelly bench, confirm after first query

### Expected Behavior
I expected jellybench to start testing

### Actual Behavior
jellybench was not successful in retrieving the necessary files to start testing

### Environment
- **OS**: Ubuntu 24.10
- **Python Version**: 3.12.7
- **CPU(s)**: Intel(R) N100
- **GPU(s)**: /
- **RAM**: 16GB DDR5

Jellyfin is not actually installed to this host.

**Comment by BotBlake** [comment]
Hi @dam-pav 
Thanks for the issue Report.

This issue is caused by the Server sending the old Source File URLs. They have Recently been Updated on a Jellyfin Repository Change.

I have reported an issue for this on the Server Repository already.
https://github.com/JPVenson/Jellyfin.HardwareVisualizer/issues/12

But since the Maintainer of the Server has been rather busy with ongoing Jellyfin Development, the Change May Take a while.

Also Note, the `Checkmarks` in the Issue Template can be `checked`. In your Case the Benchmark has NOT started yet, as the Script already failed during Setup. I will do some Work on the Issue Templates and the Wording 👌


Thank you again for Testing the Tool and taking your time to Report the issue!

**Comment by dam-pav** [comment]
Let's hope the file location issue gets resolved soon. Will retry this once I get notified. Thanks!

**Comment by BotBlake** [comment]
Hi there again @dam-pav 🫠 

I just stumpled uppon this issue and your questions about hwi.py again.

So I wanted to clear things up.
> Running hwi.py (Hardware Wallet Interface?) 

hwi.py is our own HardWare Information script.
Since there are no python modules to decently get all the neccecary hardwareinfo to run ffmpeg and deliver accurate benchmark infos on MacOS, Linus and Windows, we needed to create our own tool.


> What was it supposed to say?

hwi.py not only sends all HardwareInfo to the Main Script, but also has a standalone mode, thats mostly meant for debugging. When running it standalone, it will print all the detected hardwareinfo on the terminal.
Since we often have issues with properly detecting hardware, making sure hwi.py works and outputs correct info is a key step in troubleshooting.


> It returns: Traceback (most recent call last):
File "/home/damjan/jellybench_py/./jellybench_py/hwi.py", line 25, in
import cpuinfo
ModuleNotFoundError: No module named 'cpuinfo'

I assume this happened to you because you tried to run hwi.py outside of the poetry shell.
Note that the poetry shell is creating a python venv (a virtual python environment) in which it will install all dependencies.
This means, that inside the poetry shell, you should have all dependencies installed after running poetry install.


> Actually, the running is done with python3 (instead of python), something like this python3 ./jellybench_py/hwi.py, if that is all the same.

This here probably also is caused by running it outside the poetry shell environment. In there, python will usualy be linked to your python3 binary! 


Also, based on your issue report, I reworked both the [Known Issues](https://github.com/BotBlake/jellybench_py?tab=readme-ov-file#current-issues) section, but also our issue form!
If you dont mind, it would be a great help if you could check out the new issue form and give some feedback on it!


So far I sadly dont have any updates regarding the server situation, however I will notify you as soon as the issues are fixed👍 
Kind regards

**Comment by dam-pav** [comment]
@BotBlake 
Just so you know I noticed your reply regarding hwi two weeks ago, but I didn't feel you expected an immediate reply. Today I checked in to find if the latest PR was anyhow related to the test files issue (I don't think so).

I want to take this opportunity to thank you for your valuable hwi.py clarification. I will retry the whole procedure only after the main issue is resolved however.

As for the new issue form. To me it appears just fine; I found no trouble with the previous form, and to be honest, I don't even see what changes you made. I looks more like a survey than an issue form. Personally I would keep it simpler but I suppose you know best what questions and answers you find mandatory.

**Comment by BotBlake** [comment]
Hi @dam-pav ,  

I’m happy to share that the Server Side issues have been resolved today!  

Please remember to check the [known issues](https://github.com/BotBlake/jellybench_py?tab=readme-ov-file#current-issues) section before running.

As for the issue form:  
I’ve rephrased the wording (e.g., in the checkboxes) to clarify that all commands should be executed within the poetry shell, among other adjustments. The goal was to make it clearer what needs to be filled out.  

The general template remains mostly the same, though it’s now been moved into a GitHub form.  

Thanks and best regards!

**Comment by dam-pav** [comment]
Hi @BotBlake ,

I can confirm that the issue with the test files is resolved.
